client/src/components/EditUserForm/EditUserForm.js

```diff
@@ -19,7 +19,6 @@ export class EditUserForm extends React.Component {
       errorFirstName: '',
       errorLastName: '',
       errorUserName: '',
-      errorEmail: '',
     };
```

Comment:
@abdulmoizeng lets create a ticket for validating this on the backend (i.e. email cannot be updated)